apps/storefront/src/components/B3Tip.tsx

```diff
@@ -42,6 +42,69 @@ function MessageAlert({
   );
 }
 
+const CatalystColors = {
```

Comment:
what's the plan with all the Catalyst variants?

If there's going be lots of them, it might be worth us considering a specific folder to hold styled components which have both a standard and Catalyst versions

We could then make sure each variant adhered to the same interfaces so they can be easily interchanged in the main app

But if these are the rare exception or temporary solutions, that might be too much